core/frontend-devtools/src/tools/ChangeUnitsTool.ts

```diff
@@ -19,12 +21,21 @@ export class ChangeUnitsTool extends Tool {
   public static get minArgs() { return 0; }
   public static get maxArgs() { return 1; }
 
-  public run(useMetric?: boolean): boolean {
+  // support boolean for backwards compatibility - string allows options to four unit systems
+  public run(useMetric?: boolean | string): boolean {

```

Comment:
If you are changing what arguments a key-in accepts, please update the README accordingly.

Reply:
I reverted my change to support passing in a string as the key-in "fdt metric" would look quit odd if you keyed in "fdt metric imperial" or "fdt metric usSurvey" to set it to a different unit system.